tests/test.sh

```diff
@@ -23,6 +23,6 @@ cp ./versions.dat elm_home/0.19.0/package/versions.dat
 
 export ELM_HOME="$(pwd)"/elm_home
 
-elm make src/Main.elm --output elm.js
+elm make --debug src/Main.elm --output elm.js
```

Comment:
`npm test` still passes for me if I remove the `--debug` flag here. Can we remove it?

As I understand it, we can't run tests with `--optimize` because we need the `Debug` module, but we can use that module whether or not we have the `--debug` flag. I believe the `--debug` flags only impacts whether the debugger gets compiled in.

Reply:
oh, so dev mode still includes the toString stuff?  If so, then removing `--debug` is fine
```suggestion
elm make src/Main.elm --output elm.js
```